test/jdk/javax/swing/JRadioButton/bug4380543.java

```diff
@@ -66,10 +66,10 @@ public static void main(String[] args) throws Exception {
             public void run() {
                 try {
                     passFailJFrame = new PassFailJFrame(instructions);
-                    testObj = new testFrame();
+                    testObj = new TestFrame();
                     //Adding the Test Frame to handle dispose
-                    PassFailJFrame.addTestFrame(testObj);
-                    PassFailJFrame.positionTestFrame(testObj, PassFailJFrame.Position.HORIZONTAL);
+                    passFailJFrame.addTestFrame(testObj);
+                    passFailJFrame.positionTestFrame(testObj, PassFailJFrame.Position.HORIZONTAL);
```

Comment:
This was correct: you're calling static methods; such methods should not be called _via an instance variable_ but via the class name. Alternatively, you can static-import these methods.

Reply:
I think it wouldn't make any difference calling static methods via an instance variable or directly with class name.

Reply:
Yes, the same method would be called. Yet the difference is in the semantics: the instance variable isn't used when calling a static method, it can be null and it doesn't throw NullPointerException in this case.

The IDE raises a warning for such a usage. _You should use class name when calling static methods._ It's just cleaner: the reader would see right away it's a static method (without relying on the IDE syntax highlighting) rather than an instance method.

Please revert these two lines to using the class name to call the static methods.

Reply:
Yeah sure, updated.